examples/rest-stickies-csv/app.js

```diff
@@ -32,964 +38,273 @@ app.engine(
-                },
-              };
-              try {
-                // API Request to attach tag to sticky note
-                let response = await axios(attachConfig);
-                let attachData = JSON.stringify(response.data);
-                console.log(attachData);
-                console.log("attach url : " + attachConfig.url);
-                return attachData;
-              } catch (err) {
-                console.log(`ERROR on attachTag2(): ${err}`);
-              }
-            }
-            attachTag2();
+    const connections = pointIndexesToConnectToEachOther.map(
+      ([a, b, data]) =>
+        points[b]?.id &&
+        board.createConnector({
+          startItem: {
+            id: points[a].id,
+          },
+          endItem: {
+            id: points[b].id,
+          },
+          style,
+          ...data,
+        })
+    );
 
-            // Attach tag 3
-            async function attachTag3() {
-              let stickyId = miroData.replace(/['"]+/g, "");
-              let attachConfig = {
-                method: "post",
-                url: `https://api.miro.com/v2/boards/${process.env.boardId}/items/${stickyId}?tag_id=${tagId3}`,
-                headers: {
-                  Authorization: `Bearer ${oauthAccessToken}`,
-                  "Content-Type": "application/json",
-                },
-              };
-              try {
-                // API Request to attach tag to sticky note
-                let response = await axios(attachConfig);
-                let attachData = JSON.stringify(response.data);
-                console.log(attachData);
-                console.log("attach url : " + attachConfig.url);
-                return attachData;
-              } catch (err) {
-                console.log(`ERROR on attachTag3(): ${err}`);
-              }
-            }
-            attachTag3();
+    await Promise.all(connections);
+  };
 
-            // Attach tag 4
-            async function attachTag4() {
-              let stickyId = miroData.replace(/['"]+/g, "");
-              let attachConfig = {
-                method: "post",
-                url: `https://api.miro.com/v2/boards/${process.env.boardId}/items/${stickyId}?tag_id=${tagId4}`,
-                headers: {
-                  Authorization: `Bearer ${oauthAccessToken}`,
-                  "Content-Type": "application/json",
-                },
-              };
-              try {
-                // API Request to attach tag to sticky note
-                let response = await axios(attachConfig);
-                let attachData = JSON.stringify(response.data);
-                console.log(attachData);
-                console.log("attach url : " + attachConfig.url);
+  await createTable();
+  const stickiesAndTagsCreationPromises = createStickiesAndTags();
+  const stickiesAndTags = await Promise.all(
+    stickiesAndTagsCreationPromises
+  ).catch((error) => {
```

Comment:
since we use await and not `.then()` it's better to be consistent and use `try {} catch {}`